### ISSUE TITLE
I get a final validation performance of "VAL LOSS:  3.6068077545166015    PPL:  36.8482362494348"

### ISSUE THREAD

**Issue by darsh10** [rikdz/GraphWriter#2]
Is this good enough ? How do I run the actual evaluation.
Some of the code is absent to run the eval.py script.

**Comment by darsh10** [comment]
Unfortunately, nope.

**Comment by rikdz** [comment]
Sorry for these issues! I have update the repository with the needed code. Please reopen if more failures occur.

**Comment by Sniper970119** [comment]
Can you share your weight file? I just train for 2 epochs but it cost a whole day.
